packages/@uppy/companion-client/src/RequestClient.js

```diff
@@ -157,6 +165,13 @@ module.exports = class RequestClient {
           credentials: 'same-origin',
           body: JSON.stringify(data)
         })
+          .catch((err) => {
+            if (err.name === 'AbortError') {
+              throw err
+            } else {
+              throw new NetworkError(err)
```

Comment:
Added, what about preflight though?

Reply:
Preflight is currently implemented to [fail silently](https://github.com/transloadit/uppy/blob/master/packages/%40uppy/companion-client/src/RequestClient.js#L110-L114), to avoid CORS errors on versions of companion that don't support it. So I'd say we don't need to add the `NetworkError` there. The actual request will eventually get to catch it.